### ISSUE TITLE
Alert when underscore-prefixed variable is used?

### ISSUE THREAD

**Issue by srhb** [astro/deadnix#90]
This may be controversial, but since nix doesn't have a way to detect this (does it?) it seems easy to accidentally reintroduce a dependency on an underscore'd-by-deadnix var, and deadnix won't detect this. Perhaps that should be a failure too?

Controversy because: Now it's slightly alivenix :sweat_smile:

**Comment by astro** [comment]
In my opinion it's bad style to use "unused variables" that start with an underscore.

How about enabling this reporting with an additional command-line flag?

**Comment by srhb** [comment]
Yes, I agree, and that's what I was trying to convey: I _want_ it to warn me/fail if someone uses an underscored var. :) Flag is great!

**Comment by astro** [comment]
Please test
